packages/homepage/content/articles/codesandbox-ci-embeds/index.md

```diff
@@ -0,0 +1,95 @@
+  look for embeds.
+date: 2019-11-13
+---
+
+## CodeSandbox CI
+
+Maintaining open source projects, while enormously fulfilling, can be
+time-consuming and stressful. After a busy Hacktober, we know from first-hand
+experience what it's like, and we wanted to help library maintainers make the
+best use of their time. Today we're pleased to launch CodeSandbox CI: A free
+continuous integration service purpose-built for open source library
+maintainers.
+
+![](./images/CodeSandboxCI.gif)
+
+## Here's how it works
+
+CodeSandbox CI is a GitHub app that auto-builds your library⁠ from pull
+requests. Whenever someone opens a new pull request, CodeSandbox CI builds a new
+version of your library. Those builds get posted to our registry, so you can
+test it in CodeSandbox or locally, and all without having to publish the build
+to npm⁠. Plus, if the original GitHub issue contains a link to a reproducible
+sandbox, then we'll automatically fork it using the new PR build. That means you
+can review it online with one click.
+
+## Used by the biggest names in OSS
+
+After a successful private beta, CodeSandbox CI is already in use by projects
+like React, Babel, Framer, Material UI, and others. And it's proving to be a
+real time-saver.
+
+**Try it out for yourself by
+[installing the GitHub app](https://github.com/apps/codesandbox)**.
+
+[![Framer tweet](./images/mattgperry_tweet_1181863225756934144.png)](https://twitter.com/mattgperry/status/1181863225756934144?s=20)
+
+## A new look for Embeds
+
+CodeSandbox CI is the latest way maintainers can use CodeSandbox to save time
+and improve experiences for users and contributors. But more than 4000 open
+source projects already use CodeSandbox embeds in their docs, blog posts, and
+websites—providing visitors with working examples that help explain how to use
+their creations.
+
+Embeds provide the ability to view a sandbox inline in your content, whether you
+want to show the code, the running app, or both at the same time. Our embeds use
+a specially designed version of CodeSandbox that's small in size but almost as
+powerful as the full thing, making for a great way to explain a coding concept
+or show off a demo.
+
+We want you to be able to use embeds anywhere you want. So based on feedback
+from our community, we've launched a new default look and feel for embeds that's
+minimal in design and looks great wherever you place it. With a subtle color
+palette and no branding, it doesn't take the focus away from your great content.
+
+Danny Ruchtie, Head of Design here at CodeSandbox, notes that with this release,
+_“we've focused on building something that would better blend in with its
+surroundings while maintaining all its powerful features. So there's less focus
+on the UI and more on the content. We also wanted to give users more control and
+improve the embed experience on mobile.”_
+
+See the difference for yourself:
+[![Vanilla embed](./images/oldvsnew.png)](https://codesandbox.io/embed/vanilla)
```

Comment:
Would be cool to show a real embed here 😄

Reply:
How can we do that in markdown?

Reply:
Htnl works in markdown too :)

Reply:
This can just be the link to the sandbox and [`gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder) will handle it too.

```suggestion

https://codesandbox.io/s/vanilla
```